docker-compose.yml

```diff
@@ -1,21 +1,15 @@
 version: '3.1'
 
 services:
-
-  db:
-    image: postgres
+  pg:
+    image: postgres:12-alpine
     command:
-      - "postgres"
-      - "-c"
-      - "max_connections=1000"
-      - "-c"
-      - "shared_buffers=3GB"
+      - 'postgres'
+      - '-c'
+      - 'max_connections=100'
+      - '-c'
+      - 'shared_buffers=256MB'
```

Comment:
docker-compose is mostly for dev env => 3G is a lot on a dev env...

Reply:
Yes, your right!